react-app/src/components/DB_Trunk.js

```diff
@@ -14,46 +14,35 @@ import DB_Trunk_Cards from "./DB_Trunk_Cards";
 
 import { FaTrash as DeleteIcon } from "react-icons/fa";
 
-function DB_Trunk() {
+function DB_Trunk({ userCards }) {
   const dispatch = useDispatch();
-
   const user = useSelector((state) => state.session.user);
-  // const user_trunk_cards = useSelector((state) => state.session.user?.cards)
-  const current_trunk_card = useSelector(
-    (state) => state?.deckbuilder?.current_trunk_card
-  );
-
   const [trunkCard, setTrunkCard] = useState();
 
-  // useEffect(() => {
-  //     dispatch(getTrunkCard(current_trunk_card))
-  // }, [current_trunk_card])
-
-  const deck_id = useSelector((state) => state?.user?.decks?.id);
+  console.log(userCards);
 
   // I want to be able to click on a card and have that render within the CardView Component
 
   const handleDeleteDeck = () => {
     console.log("deleting deck...");
   };
 
-  // && user_trunk_cards
   return (
-    user && (
-      <>
-        <Box>
-          <Wrap>
-            {user.cards?.map((card) => {
-              return (
-                <WrapItem>
-                  <Box>{card}</Box>
```

Comment:
`card` in this case would be an object that contains properties of the card itself. This will cause rendering errors because we're trying to render an object, which React doesn't know how to handle